### PR TITLE
Fixes #481: Added transition effect on Student Project

### LINKED ISSUE #481: Add transition effect on Student Projects as with others.
### Expected results
Transition effect.

### Actual results
No transition effect.

### Would you like to work on this
Made a PR.

### PR DESCRIPTION
**Pull request by nzec**
- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

* Added transition effect on Student Project.
* Fixes #481 

***Preview Link : https://duttaditya18.github.io/gci17.fossasia.org/***

---
Ping @ms10398

### CODE REVIEW

**Review by anshumanv — APPROVED** [review]
LGTM

**Review by nikit19 — APPROVED** [review]

**Comment by nzec** [comment]
@anshumanv @nikit19 Thanks!

File: css/custom.css [hunk 1]
```diff
@@ -188,19 +190,18 @@ span.student-project-name {
...
   text-align: center;
   vertical-align: middle;
- }
+}
  
- .background-image-holder:hover span.student-project-name {
+.background-image-holder:hover span.student-project-name {
   opacity: 1;
   top: 0;
 }
```
Comment (vaibhavsingh97) [inline css/custom.css]: add blank line

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

File: css/custom.css (hunk 1 above)
Comment (duttaditya18) [reply css/custom.css]: There is a blank line.

Comment (nzec) [reply css/custom.css]: There is a blank line between every css block.

**Comment by arjunpandey2003** [comment]
LGTM

**Review by arjunpandey2003 — APPROVED** [review]

**Review by ParthS007 — COMMENTED** [review]
@duttaditya18 Please resolve conflicts 👍

**Comment by nzec** [comment]
@ParthS007 Done. I am actually not on my PC, so can you please squash these commits while merging?

**Review by isuruAb — COMMENTED** [review]
Please squash your commits

**Comment by nzec** [comment]
@isuruAb @ParthS007 Done!

**Comment by nzec** [comment]
@isuruAb @ParthS007 Made a new PR here : #487

**Pull request closed without merging**